sdk/monitor/monitor-opentelemetry/src/metrics/quickpulse/utils.ts

```diff
@@ -278,7 +279,10 @@ function getDependencyData(span: ReadableSpan): DependencyData {
         const dependencyUrl = new URL(String(httpUrl));
         dependencyData.Name = `${httpMethod} ${dependencyUrl.pathname}`;
       } catch (ex: any) {
-        /* no-op */
+        Logger.getInstance().info(
```

Comment:
Likewise with this one.
Another option would be an additional config to suppress these messages so that when (not if) this occurs they can see it during development but disable for production -- maybe the default is to suppress (if info is not already)

Reply:
These info logs shouldn't get noisy as the default log level is `WARN`.

Reply:
It would be worth a comment in the release notes then that this is a known issue when using NextJS that if you also set the logging level to info you will get these additional lines (soi that it's not treated as an issue)